.github/CODEOWNERS

```diff
@@ -46,9 +46,13 @@
 /experimental/dds/tree/src                     @microsoft/fluid-cr-tree
 
 # Fluid Devtools source
-/packages/tools/devtools/**/src                   @microsoft/fluid-cr-devtools
+/packages/tools/devtools/**/src                @microsoft/fluid-cr-devtools
```

Comment:
No real change here.  Just pedantically aligning the column with the other entries.